doc/classes/CanvasItem.xml

```diff
@@ -648,7 +648,7 @@
 		</signal>
 		<signal name="hidden">
 			<description>
-				Emitted when the [CanvasItem] becomes hidden, either because its own [member visible] property was set to [code]false[/code] or the [member visible] property of one of its [CanvasItem] ancestors was set to [code]false[/code].
+				Emitted when [member visible] becomes [code]false[/code] or when a change in the visibility of an ancestor hides this node (equivalent to [method is_visible_in_tree] becoming false).
```

Comment:
The new description here is incorrect, `hidden` is emitted only when the return value of `is_visible_in_tree()` changes to `false`.
(So the previous description was kinda fine.)

Changing `visible` to `false` doesn't mean it will be emitted:
![WnMoptDlM4](https://github.com/user-attachments/assets/0c74a43c-b0ff-473b-8b87-bff1fd9235f1)
![SLsW7Gamp4](https://github.com/user-attachments/assets/3798ad3c-7b85-4a16-8345-cd0bcde852a0)